docs/user/running-cincinnati.md

```diff
@@ -0,0 +1,71 @@
+# Running Cincinnati
+
+## Prerequisite
+
+APP-SRE publishes the Cincinnati container images at [Quay registry](https://quay.io/repository/app-sre/cincinnati)
+
+We need an image tag from the [Quay registry](https://quay.io/repository/app-sre/cincinnati) to use with the deployment config.
```

Comment:
Do we want to deploy by tag?  I'd rather either:

* Deploy by digest, although that would require us to bump the DeploymentConfig on each release, or
* Publish images signed by one of [Red Hat's official signing keys][1].

There's probably established precedent in the operator-marketplace / OLM space we can follow here, but I'm not sure what that predce

[1]: https://access.redhat.com/security/team/key

Reply:
The deployment config takes tag currently. I can work on fixing it to deploy using the digest. We do not need to update with every release as we can use a dummy digest similar to a dummy tag we use currently in the deployment config

Reply:
@wking  As you suggested we can just make it  `-image: ${IMAGE}` instead of `${IMAGE}:${IMAGE_TAG}` or for digest `${IMAGE}@${IMAGE_DIGEST}`. It would need changes in our e2e tests and might be some other places. So I am going to send another PR as that change is not related to this PR.